dist/marionette.component.js

```diff
@@ -1,7 +1,7 @@
 /**
- * Marionette.Component - Create and manage reusable components in Marionette.js
+ * marionette.component - Create and manage reusable components in Marionette.js
```

Comment:
Don't worry about updating the two dist files. I'll handle that with a build step.